VTEX - Marketplace APIs - Suggestions.json

```diff
@@ -2267,40 +2287,103 @@
                         }
                     }
                 }
+            },
+            "responseGetSellersApprovalSettings": {
+                "title": "GetSellersApprovalSettings",
+                "type": "object",
+                "properties": {
+                    "sellerId": {
+                        "type": "string",
+                        "description": "A string that identifies the seller in the marketplace.",
+                        "example": "sellerexample"
+                    },
+                    "accountId": {
+                        "type": "string",
+                        "description": "Marketplace’s account ID.",
+                        "example": "d74daf1f-355a-4463-bdc7-ae8b045311c"
+                    },
+                    "accountName": {
+                        "type": "string",
+                        "description": "Marketplace’s account name.",
+                        "example": "marketplaceexample"
+                    },
+                    "mapping": {
+                        "type": "object",
+                        "default": {},
+                        "description": "SKU Mapping Information and Product Specifications.\n\n The `mapping` field specifications can change according to the product type, the `color` specification is just one of the possibilities.\n\n See some examples of specifications:\n\n - Voltage \n\n - Power\n\n -Sizen\n\n - height\n\n - width",
```

Comment:
Sugestões:
- Traz links que expliquem o que são especificações, tipo essa: https://developers.vtex.com/docs/guides/catalog-overview#configuring-initial-settings-mandatory
- E sobre esse trecho `the `color` specification is just one of the possibilities.` eu acrescentaria **the example below includes properties revolving around color, but the values for those properties depend on the seller's catalog architecture, and are not default**